lib/std/jsbigints.nim

```diff
@@ -0,0 +1,159 @@
+## Arbitrary precision integers.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+when not defined(js) and not defined(nimdoc):
+  {.fatal: "Module jsbigints is designed to be used with the JavaScript backend.".}
+
+type JsBigInt* = ref object of JsRoot ## Arbitrary precision integer for JavaScript target.
+
+func newJsBigInt*(integer: SomeInteger): JsBigInt {.importjs: "BigInt(#)".} =
+  ## Constructor for `JsBigInt`.
+  runnableExamples:
+    doAssert newJsBigInt(1234567890) == newJsBigInt"1234567890"
+
+func newJsBigInt*(integer: cstring): JsBigInt {.importjs: "BigInt(#)".} =
+  ## Constructor for `JsBigInt`.
+  runnableExamples:
+    doAssert newJsBigInt"-1" == newJsBigInt"1" - newJsBigInt"2"
+
+func toCstring*(this: JsBigInt; radix: int): cstring {.importjs: "#.toString(#)".} =
```

Comment:
Maybe add the description:
`radix: An integer in the range 2 through 36 specifying the base to use for representing numeric values.`

Reply:
even better, use self documenting and self-checking code:
```nim
func toCstring*(this: JsBigInt; radix: 2..36): cstring {.importjs: "#.toString(#)".} =
  ## `radix`: base to use for representing numeric values.
```